package.json

```diff
@@ -62,7 +62,6 @@
     "sinon": "^1.14.1"
   },
   "engines": {
-    "npm": "^2.5.1",
-    "node": "^0.12.1"
+    "node": "4.2.3"
```

Comment:
it appears I patched master to use 4.2.3, and forgot to push the commit to develop